src/renderers/dom/client/__tests__/ReactRenderDocument-test.js

```diff
@@ -213,8 +228,8 @@ describe('rendering React components at document', function() {
       'quirks by rendering at the document root. You should look for ' +
       'environment dependent code in your components and ensure ' +
       'the props are the same client and server side:\n' +
-      ' (client) data-reactid=".0.1">Hello world</body></\n' +
-      ' (server) data-reactid=".0.1">Goodbye world</body>'
+      ' (client) ata-reactid=".-1.1">Hello world</body></\n' +
```

Comment:
It gets a fixed number of characters, with the addition of the minus sign it got removed from the output